examples/xray/chest_xray_image_classification.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+from forte.data.data_pack import DataPack
+from fortex.health.readers.xray_image_reader import XrayImageReader
+from fortex.health.processors.xray_image_processor import XrayImageProcessor
+from ft.onto.base_ontology import Classification
+from forte.pipeline import Pipeline
+
+import numpy as np
+from PIL import Image
+
+# default folder name
+img_folder = "sample_data"
+
+
+def main(image_pth):
+    # pipeline initialization 
+    pipeline = Pipeline[DataPack]()
+    pipeline.set_reader(XrayImageReader())
+    pipeline.add(XrayImageProcessor())
+    pipeline.initialize()
+
+    for datapack in pipeline.process_dataset(img_folder):
+        for j in datapack.get(Classification):
+            # printing the file name
+            print(datapack.pack_name)
+            # printing the classification result
+            print(j.classification_result, end="\n\n")
+
+
+if __name__ == "__main__":
+    if sys.argv[0]:
```

Comment:
made the suggested changes.

Reply:
Have you tried running the example on command line? Because it shouldn't have worked before these recent changes? Just want to ensure its correctness.

Reply:
yes, I have tried it. the sample output in read.me is the output from the code.

{'PNEUMONIA': 0.9714267253875732, 'NORMAL': 0.02857324108481407}

sample_data/normal_xray_image.jpeg
{'NORMAL': 0.9815188646316528, 'PNEUMONIA': 0.018481146544218063}

but the folder path was hard fixed so it printed the correct output. I have fixed it now, and did some additional tests to confirm all is working correctly.

Reply:
Sounds good.